tests/lib/rules/no-missing-require.js

```diff
@@ -36,6 +36,10 @@ ruleTester.run("no-missing-require", rule, {
             filename: fixture("test.js"),
             code: "require('node:fs');",
         },
+        {
+            filename: fixture("test.js"),
+            code: "require('node:test');",
+        },
```

Comment:
Added `node:test` as it's new enough that `is-builtin-module` does not have it :eyes: